README.md

```diff
@@ -4,6 +4,8 @@
 
 # openSUSE-release-tools
 
+This repository contains a set of tools to aid in the process of building, testing and releasing (open)SUSE based distributions. You can find more information about what to find in the repository in the [CONTENTS.md](CONTENTS.md) file.
```

Comment:
I am not sure but is it really only about (open)SUSE based distros? or can be it be used to any Build Service based distro?

Reply:
well, some tools are not specific to the distribution, but let's not claim we care about !SUSE distributions.